features/api.feature

```diff
@@ -72,14 +86,15 @@ Feature: DrupalContext
     And follow "First article with fields"
     Then I should see the text "PLACEHOLDER BODY"
 
+  @d7 @d8
   Scenario: Create and view a node with fields
-    Given I am viewing an "Article":
+    Given I am viewing an "article":
     | title | My article with fields! |
```

Comment:
This would be a nice followup: to allow the use of both machine names and human readable names of node types in this step definition.